cmd/kubeadm/app/cmd/version.go

```diff
@@ -22,8 +22,8 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"k8s.io/client-go/pkg/version"
```

Comment:
We should continue to use k8s.io/kubernetes/pkg/version while we are in the main repo.